Empire/Government.h

```diff
@@ -0,0 +1,118 @@
+#ifndef _Government_h_
+#define _Government_h_
+
+#include "../universe/ValueRef.h"
+#include "../util/Export.h"
+#include "../util/Pending.h"
+
+#include <boost/optional/optional.hpp>
+
+#include <memory>
+#include <string>
+#include <map>
+#include <set>
+
+
+namespace Effect {
+    class EffectsGroup;
+}
+
+class FO_COMMON_API Policy {
+public:
+    Policy(const std::string& name, const std::string& description,
+           const std::string& short_description, const std::string& category,
+           std::unique_ptr<ValueRef::ValueRef<double>>&& adoption_cost,
+           std::vector<std::unique_ptr<Effect::EffectsGroup>>&& effects,
+           const std::string& graphic);
+    ~Policy();
+
+    /** \name Accessors */ //@{
+    const std::string&  Name() const                { return m_name; }              //!< returns name of this tech
```

Comment:
Since it refers to "this tech" and this isn't a tech, I appear to have taken the contents of Tech.h and edited it into this file. I'll remove these comments as they're indeed mostly not useful, but they're not inconsistent with the style of rest of the code base, so not something I consider inherently unacceptable at present.

Reply:
When creating a new implementation it's the opportunity to get rid of baggage like that and enforce a style and not dragging old crap along.